imageroot/actions/configure-module/20config

```diff
@@ -185,6 +185,35 @@ if "timezone" in data and data["timezone"] != os.environ["WEBTOP_TIMEZONE"]:
     agent.set_env("RESTART_WEBDAV", "1")
     agent.set_env("RESTART_Z_PUSH", "1")
 
+# adjust the ejabberd module
+if "ejabberd_module" in data and data["ejabberd_module"] != os.getenv("EJABBERD_MODULE",""):
+    ejabberd_module = data["ejabberd_module"]
+    agent.set_env("EJABBERD_MODULE", ejabberd_module)
```

Comment:
Is better to move the `set_env` at the end of the `if`, just before `agent.set_env("RESTART_WEBAPP", "1")`, otherwise if something goes the result is an incomplete configuration, and no easy way to fix it.

Reply:
done